ml.md

```diff
@@ -276,6 +276,44 @@ graph IR in parts (e.g., OpenVINO stores its IR and weights separately).</p>
 <ul>
 <li><a name="method_graph_init_execution_context.0"></a> result&lt;own&lt;<a href="#graph_execution_context"><a href="#graph_execution_context"><code>graph-execution-context</code></a></a>&gt;, own&lt;<a href="#error"><a href="#error"><code>error</code></a></a>&gt;&gt;</li>
 </ul>
+<h4><a name="method_graph_list_properties"></a><code>[method]graph.list-properties: func</code></h4>
+<p>Retrieve the properties of the graph.</p>
+<p>These are metadata about the graph, unique to the graph and the
+ML backend providing it.</p>
```

Comment:
is this something supposed to be used to query zero_point/scale?

anyway the description seems too abstract. it would be nicer to give a few examples.